TelCo.ColorCoder/ColorCodeIdentifierTest.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Diagnostics;
+using System.Drawing;
+namespace TelCo.ColorCoder
+{
+    public class ColorCodeIdentifierTest
```

Comment:
Class ColorCodeIdentifierTest holds multiple test cases  but name signifies singular test
String Format and Construction process @Console.WriteLine duplicated
Main method holds all the test responsibility

Reply:
Updated the class names.
Made a separate function to format and construct string and avoided code duplication